textattack/shared/scripts/attack_args_helper.py

```diff
@@ -353,12 +422,12 @@ def parse_goal_function_and_attack_from_args(args):
             raise ValueError(f'Error: unsupported attack {args.search}')
     return goal_function, textattack.shared.Attack(goal_function, constraints, transformation, search_method)
 
-def parse_logger_from_args(args):# Create logger
+def parse_logger_from_args(args):
+    # Create logger
     attack_log_manager = textattack.loggers.AttackLogManager()
     # Set default output directory to `textattack/outputs`.
     if not args.out_dir:
-        current_dir = os.path.dirname(os.path.realpath(__file__))
-        outputs_dir = os.path.join(current_dir, os.pardir, os.pardir, os.pardir, 'outputs')
+        outputs_dir = os.path.join(os.getcwd(), 'textattack_outputs')
```

Comment:
I think the default behavior right now for non-existent output directory is to create it. I'm worried that if the user is not in the right current working directory, we'll end up confusing the users as to where they placed the outputs. I think for default outputs, we should at least be more consistent than cwd.